src/main/resources/i18n/displayStrings_hu.properties

```diff
@@ -1543,6 +1543,7 @@ WESTERN_UNION_SHORT=Western Union
 OK_PAY=OKPay
 PERFECT_MONEY=Perfect Money
 ALI_PAY=AliPay
+WECHAT_PAY=WechatPay
```

Comment:
Could you please use `WeChat Pay` instead of `WechatPay` here and also for `WECHAT_PAY_SHORT` as a translations so it matches the official website https://pay.weixin.qq.com/index.php/public/wechatpay?

Reply:
Sorry both. let me fix that. I am using Intellij..just forget to kick for 'optimize Imports' ..

my fault -_-!!